deps/wazuh_testing/wazuh_testing/vulnerability_detector.py

```diff
@@ -166,21 +188,50 @@ def insert_vulnerability(cveid = "WVE-000", target = "RHEL7", target_minor= "",
     vulnerabilities_query_string = f"""INSERT INTO VULNERABILITIES
         (cveid, target, target_minor, pending, package, operation, operation_value)
         VALUES 
-        (\"{cveid}\", \"{target}\", \"{target_minor}\", \"{pending}\", \"{package}\", \"{operation}\", \"{operation_value}\")"""
+        (\"{cveid}\", \"{target}\", \"{target_minor}\", \"{pending}\", \"{package}\", \"{operation}\",
+        \"{operation_value}\")"""
     query_vulnerabilities_info_query_string = f"""INSERT INTO VULNERABILITIES_INFO 
         (ID, title, severity, published, updated, reference, target, rationale, cvss, cvss_vector, CVSS3, 
         bugzilla_reference, cwe, advisories)
         VALUES 
-        (\"{cveid}\", \"{title}\", \"{severity}\", \"{published}\", \"{updated}\", \"{reference}\", \"{target_v}\", \"{rationale}\", \"{cvss}\",
-        \"{cvss_vector}\", \"{CVSS3}\", \"{bugzilla_reference}\", \"{cwe}\", \"{advisories}\")"""
+        (\"{cveid}\", \"{title}\", \"{severity}\", \"{published}\", \"{updated}\", \"{reference}\", \"{target_v}\",
+        \"{rationale}\", \"{cvss}\", \"{cvss_vector}\", \"{CVSS3}\", \"{bugzilla_reference}\", \"{cwe}\",
+        \"{advisories}\")"""
     make_query(CVE_DB_PATH, [vulnerabilities_query_string, query_vulnerabilities_info_query_string])
 
 
 def update_package(version, package, agent = "000"):
     """
-    Update version of installed package in databas.
+    Update version of installed package in database.
     Used to simulate upgrades and downgrades of the package given as argument.
     """
     path =  os.path.join(PACKAGES_DB_PATH, f"{agent}.db")
-    update_query_string = f"UPDATE sys_programs SET version={version} WHERE name={package};"
+    update_query_string = f"UPDATE sys_programs SET version=\"{version}\" WHERE name=\"{package}\";"
     make_query(path, [update_query_string])
+
+def delete_package(package, agent = "000"):
+    """
+    Remove package from database.
+    Used to simulate uninstall of the package given as argument or remove a package after a test.
+    """
+    path =  os.path.join(PACKAGES_DB_PATH, f"{agent}.db")
+    delete_query_string = f"DELETE FROM sys_programs WHERE name=\"{package}\";"
```

Comment:
You can simplify this expression by applying this change:
```suggestion
    delete_query_string = f'DELETE FROM sys_programs WHERE name="{package}";'
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/721/commits/a162404e20a560e8e14b7e1aa683f5b4d3d67c3e